health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -0,0 +1,242 @@
+      setShowToast({ key: "error", label: t(validDates.label) });
+      return;
+    }
+    const name = filteredCreateConfig?.[0]?.form?.[0]?.name;
+    setTotalFormData((prevData) => ({
+      ...prevData,
+      [name]: formData,
+    }));
+
+    if (formData?.CampaignName && !editName) {
+      setIsValidatingName(true);
+      let temp = await fetchValidCampaignName(tenantId, formData);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'temp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)